gips/data/core.py

```diff
@@ -704,7 +709,7 @@ def AddFile(self, sensor, product, filename, add_to_db=True):
         self.sensors[product] = sensor
         if add_to_db and orm.use_orm(): # update inventory DB if such is requested
             dbinv.update_or_add_product(driver=self.name.lower(), product=product, sensor=sensor,
-                                        tile=self.id, date=self.date, name=filename, status='complete')
```

Comment:
doesn't this look up the 'status' twice? Once here and once in the update_or_add_product function?